src/Compilers/CSharp/Portable/Symbols/SpecialTypeExtensions.cs

```diff
@@ -63,6 +63,29 @@ public static bool CanBeConst(this SpecialType specialType)
             }
         }
 
+        public static bool IsValueType(this SpecialType specialType)
```

Comment:
Actually, there is an existing `IsIntrinsicValueType` extension method in VB which also includes `SpecialType.System_DateTime` which was missing from this method. I'll move that extension method the shared assembly.